src/rpc/client.cpp

```diff
@@ -130,7 +130,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "z_importkey", 2 },
     { "z_importviewingkey", 2 },
     { "z_getpaymentdisclosure", 1},
-    { "z_getpaymentdisclosure", 2}
+    { "z_getpaymentdisclosure", 2},
+    { "z_setmigration", 0},
```

Comment:
I notice that before this list did not have a comma in anticipation of the next item but now it does. I know we do this in other languages, but I don't remember it being a standard thing in cpp. I am not sure if this could be related to the failures, but it is something that is different now.

Reply:
It is okay, allowed in cpp. It should have no relation to the failures. 

Can we wait to r-plus this again, I just wrote a unit test for this change, almost ready to force-push.